xAPI-Data.md

```diff
@@ -801,19 +802,20 @@ that domain remain unique.
 authors and/or organizations.
 * An LRS MUST NOT treat references to the same id as references to different Activities.
 * Upon receiving a Statement with an Activity Definition that differs from the one stored, an LRS
-SHOULD decide whether it considers the Activity Provider to have the authority to change the definition and
+SHOULD decide whether it considers the Learning Record Provider to have the authority to change the definition and
 SHOULD update the stored Activity Definition accordingly if that decision is positive.
-* An LRS MAY make small corrections to its canonical definition for the Activity when receiving a new definition e.g. to fix a spelling.
+* An LRS MAY make small corrections to its canonical definition for the Activity when receiving a new definition 
+e.g. to fix a spelling.
 * An LRS SHOULD NOT make significant changes to its canonical definition for the Activity based on an updated definition 
 e.g. changes to correct responses.
 
 
 ###### Activity Provider Requirements
```

Comment:
This appears to be the last remaining usage of "Activity Provider". Was this intentional? It _looks_ like it should be "Learning Record Provider".